test/parallel/test-timers-refresh.js

```diff
@@ -44,6 +44,87 @@ const { setUnrefTimeout } = require('internal/timers');
   strictEqual(timer.refresh(), timer);
 }
 
+// passing 3 arguments to setUnrefTimeout
+{
+  let called = false;
+  let result1 = '';
+  const inputArgs1 = 'a';
+  const timer = setUnrefTimeout(common.mustCall((args1) => {
+    called = true;
+    result1 = args1;
+  }), 1, inputArgs1);
+
+  setTimeout(common.mustCall(() => {
```

Comment:
This could be `common.mustNotCall`  and the timer can be assigned to a variable. Then do all the checks in the actual timer you're testing and after that  `clearTimeout`  this refed one so that it never gets called.

Reply:
@apapirovski Thanks!
I agree your opinion. I'll also fix as you recommended.

Reply:
I moved test cases to test/parallel/test-timers-setunreftimeout.js.

And in this case, callback function of this setTimeout should be called.
I didn't changed to common.mustNotCall.

Please review it again.